src/core/task.ts

```diff
@@ -15,11 +15,9 @@
  *
  */
 export class Task {
-  constructor (title: string, taskFunc: Function, skip: Function | boolean = false) {
-    return {
-      title,
-      task: taskFunc,
-      skip
-    }
+  constructor (public title: string, public task: Function, public skip: Function | boolean = false) {
+    this.title = title
+    this.task = task
+    this.skip = skip
   }
```

Comment:
```suggestion
  constructor (public title: string, public task: Function, public skip: Function | boolean = false) { }
```
I think this means the same thing?